plugins/font/plugin.js

```diff
@@ -102,27 +108,35 @@
 				editor.fire( 'saveSnapshot' );
 
 				var previousValue = this.getValue(),
-					style = styles[ value ];
+					style = styles[ value ],
+					previousStyle,
+					range,
+					path,
+					matching,
+					startBoundary,
+					endBoundary,
+					node,
+					bm,
+					defaultValue = 'cke-default';
```

Comment:
Same default value is already defined in an `init` method, maybe it could be extracted somewhere to be more "global"? Especially that it is used also in other places (tests).